src/assets/styles/_base.scss

```diff
@@ -8,13 +8,16 @@ $palette-five: #481620;
 $default-padding: 5px;
 $default-horizontal-padding: 5px;
 $default-vertical-padding: 5px;
-$default-card-height: 150px;
-$default-card-width: 100px;
-$in-play-width: $default-card-width * 7; 
-$in-play-height: $default-card-height*2;
+$game-width: 1024px;
+$game-height: 768px;
+$card-height: 110px;
```

Comment:
Not all SASS vars need to go in `_base.scss`, just vars that need to be imported into every file. `$card-height` and `$card-width` should be defined in `_card.scss`, `$button-width` can be defined in `_button.scss`, etc.